contracts/Rivendale.sol

```diff
@@ -132,9 +153,19 @@ contract Rivendale {
                 voteBreakdown[_id].walkerTally++;
             }
         }
-        if (voteBreakdown[_id].walkerCount > 0){
-            weightedVotes[0] = weights.walkerWeight * (voteBreakdown[_id].walkerTally / voteBreakdown[_id].walkerCount);
+
+        // Since walkers are a priviledged group, shouldn't the tally be based on the total amount of walkers? If there's huge abstention, a single walker can have 40% of voting power.
+
+        // Follow up if above is implemented:
+        // If a waker is banished while a vote is open, will this calculation be messed up?
+        if (voteBreakdown[_id].walkerCount > 0) {
+            weightedVotes[0] =
+                weights.walkerWeight *
+                (voteBreakdown[_id].walkerTally /
+                    voteBreakdown[_id].walkerCount);
         }
+
+        // Can't rich members, in a way, pay to have larger weight passing as a payee? This could be an attack vector in periods of few paying customers
         //increment payee contribution total by voter's contribution
         voteBreakdown[_id].payeeCount += _fellowship.payments(msg.sender);
         //should we make this just "balanceOf" to make it ERC20 compliant

```

Comment:
thoughts?

Reply:
Hmmm I think with balanceOf we can have double votes with the same balance.